ci/docker/build/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ubuntu:14.04
+
+ENV DEPOT_TOOLS_PATH $HOME/depot_tools
+ENV ENGINE_PATH $HOME/engine
+
+RUN apt-get update
+RUN apt-get install -y git wget curl unzip python lsb-release sudo
+
+RUN git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git $DEPOT_TOOLS_PATH
+ENV PATH $PATH:$DEPOT_TOOLS_PATH
+
+RUN mkdir --parents $ENGINE_PATH
+WORKDIR $ENGINE_PATH
+ADD engine_gclient .gclient
+RUN gclient sync
+
+WORKDIR $ENGINE_PATH/src
+RUN gclient sync
```

Comment:
Why do you need to run this again?  If the .gclient and DEPS files didn't change, then running it again shouldn't do anything.